spec/lib/camt_parser/052/report_spec.rb

```diff
@@ -15,4 +15,7 @@
   specify { expect(ex_rpt.closing_balance).to be_kind_of(CamtParser::AccountBalance) }
 
   specify { expect(ex_rpt.identification).to eq("0352C5220131227110203") }
+
+  specify { expect(ex_rpt.from_date_time).to be_kind_of(Time) }
```

Comment:
Would you mind also testing that the value was parsed correctly?
I would also like to see the `nil` case tested, which is handled in the method.

Reply:
Added a check for the `Time` value.

As for the `nil` case, what would be the best way to implement it?

Reply:
OK, I kept the original fixture intact - the `nil` checks run against that, and added a new fixture for parsing checks. I used my (anonymized/changed) bank export...